docs/how-to/topics/file-management.rst

```diff
@@ -0,0 +1,206 @@
+
+Absolute paths are the full path to a location, constructed using the root directory of the file system as the anchor (``/`` on macOS and Linux, and ``C:\`` on Windows). They do not rely on the location of the Python program or app, and they are valid from anywhere in the file system. At runtime, a relative path is resolved to an absolute path, constructed using the current working directory as the anchor.
+
+Relative paths might work with simple Toga scripts. However, when you get into packaging applications, things change. When you double-click on an icon to run an app, what directory is considered the current working directory? Is it the directory containing the app bundle? Is it the directory in the app bundle that holds the executable? There isn't a solid answer. The question we really need to answer is, how can we reliably tell an app where to look for file content?
+
+If we can't rely on the current working directory to construct relative paths, what *can* we use? We need to use absolute paths.
+
+Absolute paths
+==============
+
+There are multiple ways to construct absolute paths in Python.
+
+The most basic method is to spell out the entire path. For example, on macOS or Unix, to access ``file.txt`` in a directory called ``file_directory`` in your home directory, you could use ``~/file_directory/file.txt``.
+
+In any Python file, ``__file__`` resolves to the absolute path to the location of file being executed, provided as a string. We can use this to construct paths; for example, ``Path(__file__).parent`` is the parent directory of the currently running Python program.
+
+The ``pathlib`` module, from the Python standard library, provides several absolute path options including ``Path.cwd()``, which is the path to the current working directory, and ``Path.home()``, which is the path to your home directory.
+
+Let's take a look at an example of reading file contents from a file using the basic method to construct an absolute path.
+
+Create a new Briefcase project (for a refresher on how to do this, see `the BeeWare tutorial <https://docs.beeware.org/en/latest/tutorial/tutorial-1.html>`__). Once you've created that project, update ``app.py`` in the project to contain the following:
+
+.. code-block:: python
+
+    from pathlib import Path
+
+    import toga
+    from toga.style.pack import COLUMN
+
+
+    class ConfigFileCreator(toga.App):
+        def startup(self):
+            self.text_input = toga.MultilineTextInput()
+            self.config_directory = toga.TextInput(readonly=True)
```

Comment:
Minor nit - might want to name this `config_directory_input`, so it doesn't read as if it *is* the config directory.

Reply:
(or a similar, shorter name...)